test/e2e/gardener/shoot/internal/node/node.go

```diff
@@ -35,79 +34,81 @@ const waitForCSINodeAnnotation = v1beta1constants.AnnotationPrefixWaitForCSINode
 const driverName = "foo.driver.example.org"
 
 // VerifyNodeCriticalComponentsBootstrapping tests the node readiness feature (see docs/usage/advanced/node-readiness.md).
-func VerifyNodeCriticalComponentsBootstrapping(parentCtx context.Context, f *framework.ShootFramework) {
-	ctx, cancel := context.WithTimeout(parentCtx, 15*time.Minute)
-	defer cancel()
-
-	shootClientSet, err := access.CreateShootClientFromAdminKubeconfig(ctx, f.GardenClient, f.Shoot)
-	ExpectWithOffset(1, err).To(Succeed())
-
-	var (
-		seedClient  = f.SeedClient.Client()
-		shootClient = shootClientSet.Client()
-		technicalID = f.Shoot.Status.TechnicalID
-	)
-
-	By("Create ManagedResources for shoot with broken node-critical components")
-	createOrUpdateNodeCriticalManagedResource(ctx, seedClient, shootClient, technicalID, nodeCriticalDaemonSetName, "non-existing", false)
-	createOrUpdateNodeCriticalManagedResource(ctx, seedClient, shootClient, technicalID, csiNodeDaemonSetName, "non-existing", true)
-	DeferCleanup(func() {
-		cleanupCtx, cancel := context.WithTimeout(context.Background(), time.Minute)
-		defer cancel()
-
-		cleanupNodeCriticalManagedResource(cleanupCtx, seedClient, technicalID, nodeCriticalDaemonSetName)
-		cleanupNodeCriticalManagedResource(cleanupCtx, seedClient, technicalID, csiNodeDaemonSetName)
+func VerifyNodeCriticalComponentsBootstrapping(s *ShootContext) {
+	GinkgoHelper()
+
+	var technicalID string
```

Comment:
Nit: in the context of GEP-28, we want to distinguish between technical ID and seed namespace, ref https://github.com/gardener/gardener/pull/11382
```suggestion
	var seedNamespace string
```